commonItems/BufferedReader.cs

```diff
@@ -12,14 +12,14 @@ namespace commonItems {
     /// but later updated
     /// </summary>
     public class BufferedReader {
-        private readonly StreamReader streamReader;
+        public StreamReader StreamReader { get; }
```

Comment:
You have the variable with the same capitalization as the type? Seems likely to cause confusion down the line.

Reply:
BaseStreamReader?